tests/integration/test.prefix.js

```diff
@@ -38,7 +38,9 @@ describe('test.prefix.js', function () {
 if (typeof process !== 'undefined' &&
     !process.env.LEVEL_ADAPTER &&
     !process.env.LEVEL_PREFIX &&
-    !process.env.ADAPTER) {
+    !process.env.ADAPTER &&
+    // fails on windows with EBUSY - "resource busy or locked", not worth fixing
```

Comment:
No probs, just thought I'd share since you're now in the (lovely?) world of win dev ;)